landing-pages/index.html

```diff
@@ -147,7 +161,8 @@ <h3>Synonym Finder</h3>
                 <a href="/web-ai-demos/ai-synonym-finder/">Open</a>
             </div>
         </div>
-        <h2>Client side AI Demos</h2>
+        <h2 id="clientside">Client-side AI demos</h2>
+        <p>You can 
```

Comment:
Looks like this bit ended up incomplete.

Reply:
Nooooooo